src/controllers/spa.controller.js

```diff
@@ -13,3 +15,55 @@ exports.getAllServices = (req, res, next) => {
         });
     });
 }
+
+exports.createService =  async (req, res) => {
+    console.log("Create service controller");
+
+    const { title, descript } = req.body;
+
+    if (!title || !descript) {
+        return res.status(400).json({
+          status: 0,
+          message: "Both 'title' and 'descript' are required fields.",
+        });
+    }
+
+    spaService.createService(title, descript, (error, result) => {
+        if(error){
+           return res.status(400).json({
+                status: 0,
+                message: err.message
+            });
+        }
+        return res.status(200).json({
+            status: 1,
+            message: result.recordsets[0]
+        })
+    })
+};
+
+
+exports.deleteService =  (req, res) => {
+
+    const id = req.body.id;

```

Comment:
When using an id for DELETE or GET, it's customary for ID to be in the query rather than the body. So that would be req.query.id. And then when sending the request, put the id in the URL.